gepetuto/__main__.py

```diff
@@ -67,6 +74,19 @@ def get_tp_id():
         current_tp_id += 1
 
 
+def retrieve_python_version():
+    """Retrieve installed python version."""
+    for path in sys.path:
```

Comment:
`sys.path` ? This is a list of directories where we should have python modules. The intepreter is provided at `sys.executable`.

If your goal is to find if there is an executable `python` or `python3`, I guess we can simply use a try/catch with a check_call on `python --version`: if it raise a `FileNotFoundError`, this is not a valid interpreter.

Reply:
yes I was looking for the executable, but if we use `python --version`,  won't we will still not know if we should use `python` or `python3` ?